### ISSUE TITLE
SunCertPathBuilderException: unable to find valid certification path to requested target

### ISSUE THREAD

**Issue by bsirang** [silvergasp/bazel-embedded#40]
I started receiving this build failure today. This is running in a github workflow which had been fine for weeks up until today.

```
INFO: Repository com_gcc_arm_none_eabi_compiler instantiated at:
  /github/workspace/WORKSPACE:186:22: in <toplevel>
  /github/home/.cache/bazel/_bazel_root/3ef6b2ad9cc100f006a5d4a7217ff063/external/bazel_embedded/toolchains/compilers/gcc_arm_none_eabi/gcc_arm_none_repository.bzl:89:28: in gcc_arm_none_compiler
Repository rule gcc_arm_none_repository defined at:
  /github/home/.cache/bazel/_bazel_root/3ef6b2ad9cc100f006a5d4a7217ff063/external/bazel_embedded/toolchains/compilers/gcc_arm_none_eabi/gcc_arm_none_repository.bzl:77:42: in <toplevel>
WARNING: Download from https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=6e63531f-8cb1-40b9-bbfc-8a57cdfc01b4&la=en&hash=F761343D43A0587E8AC0925B723C04DBFB848339 failed: class javax.net.ssl.SSLHandshakeException PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
ERROR: An error occurred during the fetch of repository 'com_gcc_arm_none_eabi_compiler':
   Traceback (most recent call last):
	File "/github/home/.cache/bazel/_bazel_root/3ef6b2ad9cc100f006a5d4a7217ff063/external/bazel_embedded/toolchains/compilers/gcc_arm_none_eabi/gcc_arm_none_repository.bzl", line 30, column 40, in _com_gcc_arm_none_repository_impl
		repository_ctx.download_and_extract(
Error in download_and_extract: java.io.IOException: Error downloading [https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=6e63531f-8cb1-40b9-bbfc-8a57cdfc01b4&la=en&hash=F761343D43A0587E8AC0925B723C04DBFB848339] to /github/home/.cache/bazel/_bazel_root/3ef6b2ad9cc100f006a5d4a7217ff063/external/com_gcc_arm_none_eabi_compiler/temp17661210164835272010/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
ERROR: Error fetching repository: Traceback (most recent call last):
	File "/github/home/.cache/bazel/_bazel_root/3ef6b2ad9cc100f006a5d4a7217ff063/external/bazel_embedded/toolchains/compilers/gcc_arm_none_eabi/gcc_arm_none_repository.bzl", line 30, column 40, in _com_gcc_arm_none_repository_impl
		repository_ctx.download_and_extract(
Error in download_and_extract: java.io.IOException: Error downloading [https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=6e63531f-8cb1-40b9-bbfc-8a57cdfc01b4&la=en&hash=F761343D43A0587E8AC0925B723C04DBFB848339] to /github/home/.cache/bazel/_bazel_root/3ef6b2ad9cc100f006a5d4a7217ff063/external/com_gcc_arm_none_eabi_compiler/temp17661210164835272010/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
ERROR: no such package '@com_gcc_arm_none_eabi_compiler//': java.io.IOException: Error downloading [https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=6e63531f-8cb1-40b9-bbfc-8a57cdfc01b4&la=en&hash=F761343D43A0587E8AC0925B723C04DBFB848339] to /github/home/.cache/bazel/_bazel_root/3ef6b2ad9cc100f006a5d4a7217ff063/external/com_gcc_arm_none_eabi_compiler/temp17661210164835272010/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

I'm unfamiliar with how Bazel + JDK manage certificates, so I'm not entirely sure if this is an issue specific to the bazel-embedded rules.

**Comment by silvergasp** [comment]
This is a problem where ARM changed its URLs. This will be fixed with #42.

**Comment by silvergasp** [comment]
Sorted! Thanks for raising this.
